units/systemd-timesyncd.service.in

```diff
@@ -12,7 +12,8 @@ ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
 RequiresMountsFor=/var/lib/systemd/clock
-After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service
+Requires=dev-rtc.device
+After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service dev-rtc.device
```

Comment:
Hm, when implementing inside the Manager I get Permission denied... I guess this is because we already dropped privileges required to read/write the RTC...